backend/src/test/java/com/benine/backend/http/RequestHandlerTest.java

```diff
@@ -42,6 +42,29 @@ public final void testRespond() throws Exception {
     verify(out).close();
   }
 
+  @Test
```

Comment:
You could move the creation of a mock camera, handler and exchange to an `@before` method, reducing duplicate code.